src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDeclarationTests.vb

```diff
@@ -684,10 +684,11 @@ Property Scen4(
 p1 as vb$anonymous1
 ) a
 ]]>,
-            Diagnostic(ERRID.ERR_InvalidParameterSyntax, "anonymous1"),
```

Comment:
+1, Not sure why this code is impacted by this change.

---
In reply to: [63097125](https://github.com/dotnet/roslyn/pull/11180#discussion_r63097125) [](ancestors = 63097125)